ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -69,6 +70,32 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P6S Exocleaver',
+      // Unholy Darkness stack headmarkers are same time as first Exocleaver
+      // Exchange of Agonies headmarkers are 3s before second Exocleavers
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['7869', '786B'], source: 'Hegemone', capture: false }),
+      alertText: (data, _matches, output) => {
+        if (data.secondExocleavers)
+          return output.protean!();
+        return output.healerGroupsProtean!();
+      },
+      run: (data) => data.secondExocleavers = true,
+      outputStrings: {
+        healerGroupsProtean: {
+          en: 'Healer Groups + Protean',
+        },
+        protean: {
+          en: 'Protean',
```

Comment:
This is kind of an extreme nit here about wording, but I think this is not really a protean in the sense that the boss is shooting a conal at every player.  When I think protean, I think "people need to be in clock positions".  I think if you wanted to prompt the players here, you could do a "Healer Groups" but then do a Responses.moveAway() after the first hit of the excocleaver to remind people not to plant.

Reply:
Makes sense.